packages/web-client/app/components/card-pay/deposit-workflow/transaction-status/index.hbs

```diff
@@ -2,36 +2,46 @@
   @header="Token Bridge"
   @isComplete={{@isComplete}}
 >
-  <Boxel::ProgressSteps
-    @progressSteps={{this.progressSteps}}
-    @completedCount={{this.completedCount}}
-    as |progressStep|
-  >
-    <span
-      data-test-token-bridge-step={{progressStep.index}}
-      data-test-completed={{progressStep.completed}}
+  <div class="transaction-status">
+    <p class="transaction-status__description">
+      Bridging tokens to xDai blockchain
+    </p>
+
+    <Boxel::ProgressSteps
+      class="transaction-status__steps"
+      @progressSteps={{this.progressSteps}}
+      @completedCount={{this.completedCount}}
+      as |progressStep|
     >
-      {{progressStep.title}}
-    </span>
+      <span
+        class="transaction-status__step-title"
+        data-test-token-bridge-step={{progressStep.index}}
+        data-test-completed={{progressStep.completed}}
+      >
+        {{progressStep.title}}
+      </span>
 
-    {{#if (eq progressStep.index 0)}}
-      <Boxel::Button @as="anchor" @size="extra-small" @href={{this.depositTxnViewerUrl}} target="_blank" data-test-etherscan-button>
-        View on Etherscan
-      </Boxel::Button>
-    {{/if}}
-    {{#if (eq progressStep.index 1)}}
-      <Boxel::Button @as="anchor" @size="extra-small" @href={{this.bridgeExplorerUrl}} target="_blank" data-test-bridge-explorer-button>
-        View on Bridge Explorer
-      </Boxel::Button>
-    {{/if}}
-    {{#if (eq progressStep.index 2)}}
-      {{#if progressStep.completed}}
-        <Boxel::Button @as="anchor" @size="extra-small" @href={{this.blockscoutUrl}} target="_blank" data-test-blockscout-button>
-          View on Blockscout
-        </Boxel::Button>
-      {{else}}
-        Processing...
-      {{/if}}
-    {{/if}}
-  </Boxel::ProgressSteps>
+      <span class="transaction-status__step-status">
+        {{#if (eq progressStep.index 0)}}
+          <Boxel::Button @as="anchor" @size="extra-small" @href={{this.depositTxnViewerUrl}} target="_blank" data-test-etherscan-button>
+            View on Etherscan
+          </Boxel::Button>
+        {{/if}}
+        {{#if (eq progressStep.index 1)}}
+          <Boxel::Button @as="anchor" @size="extra-small" @href={{this.bridgeExplorerUrl}} target="_blank" data-test-bridge-explorer-button>
+            View on Bridge Explorer
+          </Boxel::Button>
```

Comment:
not sure if this is necessary these days - but do we need to add `rel="noopener"` to links with `target="_blank"`?

Reply:
https://mathiasbynens.github.io/rel-noopener/

Seems like most browsers we plan to support have decided that `rel="noopener"` is implicit when `target="_blank"` this year.

Reply:
Not sure about their previous versions though, or if that matters